web/styles/input_pill.css

```diff
@@ -27,7 +27,6 @@
         &:focus {
             color: hsl(0deg 0% 100%);
             border: 1px solid hsl(176deg 78% 28%);
-            background-color: hsl(176deg 49% 42%);
```

Comment:
This change should be in its own commit with its own commit message, since it's not directly part of adding the double-click behavior.